gfeloqua.class.php

```diff
@@ -1610,20 +1608,28 @@ function update_retry_attempts( $entry_id, $form_id = NULL ){
 		gform_update_meta( $entry_id, GFELOQUA_OPT_PREFIX . 'retries', $attempts, $form_id );
 	}
 
-	function add_success_meta( $meta, $form_id ){
-		$feeds = GFAPI::get_feeds( NULL, $form_id, $this->_slug );
+	/**
+	 * Add the "Sent to Eloqua" entry meta property.
+	 *
+	 * @param array $entry_meta An array of entry meta already registered with the gform_entry_meta filter.
+	 * @param int $form_id The id of the current form.
+	 *
+	 * @return array
+	 */
+	public function get_entry_meta( $entry_meta, $form_id ) {
+		$feeds = $this->get_feeds();
 
-		if( ! $feeds )
-			return $meta;
+		if ( ! $feeds ) {
+			return $entry_meta;
+		}
 
-		$meta[ GFELOQUA_OPT_PREFIX . 'success'] = array(
-			'label' => __( 'Sent to Eloqua?', 'gfeloqua' ),
-			'is_numeric' => false,
-			'update_entry_meta_callback' => 'update_entry_meta',
+		$meta[ GFELOQUA_OPT_PREFIX . 'success' ] = array(
```

Comment:
👍  Thanks, I see it now. Sorry, this is my first time dealing with a PR, so bear with me!